config/environments/staging.rb

```diff
@@ -86,4 +87,4 @@
 end
 
 # Required when using Rails.application.routes.url_helpers from outside the request/response life cycle (models, jobs, lib)
-Rails.application.routes.default_url_options = { host: 'era-test.library.ualberta.ca', port: 443, protocol: 'https' }
+Rails.application.routes.default_url_options = { host: Rails.application.secrets.default_url_host, port: Rails.application.secrets.default_url_port, protocol: 'https' }
```

Comment:
Not sure you would ever change the default url port here (from 443), but i guess this doesnt hurt